### ISSUE TITLE
Stop scoolling and draw over there

### ISSUE THREAD

**Issue by diegoceldranmorell** [barteksc/AndroidPdfViewer#484]
Hi, i'm trylling STOP manually the scrooll function but it's not working using
setVerticalScrollBarEnabled(false);
setHorizontalScrollBarEnabled(false);
methods.. so i can draw over a page overriding the onTouchEvent(MotionEvent) {}

**Comment by 1stmetro** [comment]
overwrite the methods and create your own,

What do you mean by draw over? as in a watermark ? 

If you do then this is possible without stopping scrolling.

**Comment by diegoceldranmorell** [comment]
No no no, after loading, print an image in absolute position over a page, after that, save the new file in Device Memory

**Comment by 1stmetro** [comment]
This can all be done without using source as I do this myself.

 It's just a case of using alpha on the overlaying image.

**Comment by diegoceldranmorell** [comment]
Really? How?

**Comment by 1stmetro** [comment]
Not near a pc but which bit are you stuck on?

**Comment by diegoceldranmorell** [comment]
Actually, i'm using an invisible canvas over the PDFViewer, when y clic a button, i active the invisible canvas, making posible draw over there, i wan't to take the position where i'm drawing, so, detected the actually relative position in the actually page, and from a PNG image generated via the invisible canvas, attach this png in a new PDF file, after, refresh the pdfviewer loading the new PDF file with the actually zoom at the pdfviewer

**Comment by 1stmetro** [comment]
Sorry if i'm reading this wrong you want to overlay a png file and then save the pdf with the overlay included in the new pdf ?

**Comment by 1stmetro** [comment]
This is a sample of how to use an overlay image, in this case its an overlay displayed on a single page or in my case every page as a watermark but i dont use this image.   This is done using pdfviewer and no other layouts over the top, etc.

![number](https://user-images.githubusercontent.com/21983778/34394821-d4dddcc0-eb53-11e7-8773-79eb8419c207.png)

**Comment by diegoceldranmorell** [comment]
Really, if i can choose where i wan't to put the png file and select a 100% opacity for this png file, and later, save it like a new pdf it's perfect to me, it's possible?
![image](https://user-images.githubusercontent.com/11218421/34395838-10508216-eb64-11e7-865a-12fc1e970fc0.png)

How you can see, use this to create a new pdf file attaching the sign, (saved in a png file).
To this purpose i use an invisible canvas over there, to draw a sign in a position in a empty png file, when accept the sign i save the sign in a png file and refresh the pdfviewer attaching the new sign in the correct position and save it in the correct position, also, i can attach in this position the png file and later load the new signed pdf file with a normal pdf file.

**Comment by 1stmetro** [comment]
i dont believe you can save pdf's using this package.... but you could create an image of the combined as a document...

Never looked into saving.

**Comment by 1stmetro** [comment]
Saving of the image after a signature as a pdf can be saved using itext.
